apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/CompiledGraphQL.kt

```diff
@@ -154,6 +154,12 @@ sealed class CompiledNamedType(val name: String) : CompiledType() {
   override fun leafType() = this
 }
 
+/**
+ * A GraphQL scalar type that is mapped to a Kotlin. This is named "Custom" for historical reasons
+ * but is also used for builtin scalars
+ * 
+ * TODO v4: rename this to ScalarType
```

Comment:
I started adding "TODO v4" for stuff that will need to be reworked. I don't think we had any marker previously?

Reply:
Good idea!
(I don't think we did)